roles/network_plugin/cilium/templates/cilium/ds.yml.j2

```diff
@@ -96,7 +96,7 @@ spec:
               fieldPath: metadata.namespace
         - name: CILIUM_CLUSTERMESH_CONFIG
           value: /var/lib/cilium/clustermesh/
-{% if cilium_kube_proxy_replacement == 'strict' %}
+{% if (cilium_kube_proxy_replacement == 'strict') or cilium_kube_proxy_replacement %}
```

Comment:
Will there be an implicit conversion problem with the latter condition?

I just used online jinja2 parser to test other strings and it will be true.

Reply:
ah well spotted, that should really be an explicit bool true check, and an explicit string true check, otherwise we'd catch 'false' & 'partial' strings too...

Reply:
pushed there, checks seem ok, other than the usual 429 again :(